homeassistant/components/prusalink/camera.py

```diff
@@ -35,7 +35,11 @@ def __init__(self, coordinator: JobUpdateCoordinator) -> None:
     @property
     def available(self) -> bool:
         """Get if camera is available."""
-        return super().available and self.coordinator.data.get("job") is not None
+        return (
+            super().available
+            and (file := self.coordinator.data.get("file"))
+            and file.get("refs").get("thumbnail")
```

Comment:
```suggestion
            and file.get("refs", {}).get("thumbnail")
```

To prevent raising an AttributeError as `None` has no `get` function